### PR TITLE
feat(plugins/Collectible): support previewing video NFT

### PR DESCRIPTION
**Pull request by UncleBill**
closes #

### CODE REVIEW

**Review by guanbinrui — DISMISSED** [review]

File: packages/maskbook/src/plugins/Collectible/UI/ArticleTab.tsx [hunk 1]
```diff
@@ -8,29 +11,50 @@ const useStyles = makeStyles((theme) => {
...
+}
+
 export interface ArticleTabProps {}
 
 export function ArticleTab(props: ArticleTabProps) {
     const classes = useStyles()
     const { asset } = CollectibleState.useContainer()
 
     if (!asset.value) return null
+    const resourceUrl = asset.value.image_url || asset.value?.animation_url
```
Comment (zhouhanseng) [inline packages/maskbook/src/plugins/Collectible/UI/ArticleTab.tsx:48]:
```suggestion
const resourceUrl = asset.value.image_url || asset.value.animation_url
```

File: packages/maskbook/src/plugins/Collectible/UI/ArticleTab.tsx [hunk 2]
```diff
@@ -8,29 +11,50 @@ const useStyles = makeStyles((theme) => {
...
+
+// opensea support: JPG, PNG, GIF, SVG, MP4, WEBM, MP3, WAV, OGG, GLB, GLTF.
+const AssetPlayer: FC<AssetPlayerProps> = ({ src, alt }) => {
+    const classes = useStyles()
+    if (!src) {
+        return null
+    }
+    const isVideo = src.match(/\.(mp4|webm)$/i)
+    if (isVideo) {
+        return <Video src={src} VideoProps={{ className: classnames(classes.player) }} />
```
Comment (zhouhanseng) [inline packages/maskbook/src/plugins/Collectible/UI/ArticleTab.tsx:35]: unnecessary `classnames`?

Comment (zhouhanseng) [inline packages/maskbook/src/plugins/Collectible/UI/ArticleTab.tsx:35]: unnecessary `classnames`?

Comment (zhouhanseng) [inline packages/maskbook/src/plugins/Collectible/UI/ArticleTab.tsx:35]: Unnecessary `classnames`?

**Review by zhouhanseng — APPROVED** [review]

**Review by Jack-Works — APPROVED** [review]

**Pull request merged**